cmake/SDL_ttf_Config.cmake.in

```diff
@@ -0,0 +1,15 @@
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+
+if(NOT TARGET freetype::freetype)
```

Comment:
Use same `find_package` calls:
```
find_package(freetype CONFIG REQUIRED)
find_package(SDL2 CONFIG REQUIRED)
```

`find_dependency` designed to be used here but it doesn't work with `CONFIG` correctly. `find_package` fits fine in our scenario.